CHANGES

```diff
@@ -9,6 +9,13 @@
 
  Changes between 1.1.0h and 1.1.1 [xx XXX xxxx]
 
+  *) Change generating and checking of primes so that the error rate of not
+     being prime depends on the intended use based on the size of the input.
+     For larger primes this will result in more rounds of Miller-Rabin.
+     The maximal error rate for primes with more than 1080 bits is lowered
+     to 2^-128.
+     [Kurt Roeckx]
```

Comment:
`[Kurt Roeckx, Annie Yousar]`

At the risk of getting on your nerves: IMHO Annie contributed a significant part to this commit and deserves to be mentioned.

Reply:
Ditto to included Annie in the changes entry.
Approval reconfirmed too.